miso-web/src/main/webapp/pages/editArray.jsp

```diff
@@ -0,0 +1,201 @@
+<%--
+  ~ Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+
+  ~ MISO project contacts: Robert Davey @ TGAC
+  ~ **********************************************************************
+  ~
+  ~ This file is part of MISO.
+  ~
+  ~ MISO is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ MISO is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+  ~
+  ~ **********************************************************************
+  --%>
+<%@ include file="../header.jsp" %>
+<script src="<c:url value='/scripts/jquery/datatables/js/jquery.dataTables.min.js'/>" type="text/javascript"></script>
+<script src="<c:url value='/scripts/jquery/editable/jquery.jeditable.mini.js'/>" type="text/javascript"></script>
+<script src="<c:url value='/scripts/jquery/editable/jquery.jeditable.datepicker.js'/>" type="text/javascript"></script>
+<script src="<c:url value='/scripts/jquery/editable/jquery.jeditable.checkbox.js'/>" type="text/javascript"></script>
+<link href="<c:url value='/scripts/jquery/datatables/css/jquery.dataTables.css'/>" rel="stylesheet" type="text/css">
+<link rel="stylesheet" href="<c:url value='/scripts/jquery/datatables/css/jquery.dataTables_themeroller.css'/>">
+
+<div id="maincontent">
+<div id="contentcolumn">
+
+<div id="tab-1">
+
+<form:form id="array-form" data-parsley-validate="" autocomplete="off" acceptCharset="utf-8">
+<h1>
+  <c:choose>
+    <c:when test="${pageMode eq 'create'}">Create</c:when>
+    <c:otherwise>Edit</c:otherwise>
+  </c:choose> Array
+  <button id="save" type="button" class="fg-button ui-state-default ui-corner-all" onclick="SampleArray.validateAndSave()">Save</button>
+</h1>
+
+<div class="bs-callout bs-callout-warning hidden">
+  <h2>Oh snap!</h2>
+  <p>This form seems to be invalid</p>
+  <div id="generalErrors"></div>
+</div>
+<br/>
+
+<div id="arrayInfo">
+  <table class="in">
+    <tr>
+      <td class="h">Array ID:</td>
+      <td><span id="id">Unsaved</span></td>
+    </tr>
+    <tr>
+      <td class="h">Array Model:*</td>
+      <td>
+      <c:choose>
+        <c:when test="${pageMode eq 'create'}">
+          <select id="arrayModel">
```

Comment:
Can this be a `<miso:select>`?

Reply:
I tried, but `<miso:select>` needs to be bound to a model attribute and this form uses JS to load data to/from JSON for AJAX instead